src/styles/card.scss

```diff
@@ -0,0 +1,112 @@
+.card {
+  width: 166px;
+  height: 358px;
+  background-color: #fff;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  font-family: Roboto, sans-serif;
```

Comment:
The font-family should be set in the root or body selector and inherited by the rest of the elements. This way, if you need to change the font later, you won't have to do it in multiple places.